data/dapps/orbit-protocol.json

```diff
@@ -0,0 +1,7 @@
+{
+  "aliases": {
+    "orbit": ["blast"]
+  },
+  "name": "Orbit Protocol",
```

Comment:
These guys for example call themselves Orbit everywhere except Twitter so I went with the version without fluff